test/e2e_node/pids_test.go

```diff
@@ -128,5 +128,6 @@ var _ = SIGDescribe("PodPidsLimit", framework.WithSerial(), func() {
 			initialConfig.PodPidsLimit = int64(1024)
 		})
 		runPodPidsLimitTests(f)
+		cleanupPods(f)
```

Comment:
should we clean up before the test or after? Looks inconsistent with what is doing in swap

Reply:
See above, renaming the method. There is an AfterEach added in this method which kicks in later